### PR TITLE
Refactor get-users to publish to Kinesis stream instead of SNS

### PR DESCRIPTION
**Pull request by phamann**
Replaces the publishing to SNS with Kinesis.

### CODE REVIEW

**Comment by phamann** [comment]
Whitespace is borked due to my new editor config... will sort out today. Need to ship to confirm with operations that the new IAM rules are working correctly.

File: functions/get-users/index.js [hunk 1]
```diff
@@ -5,44 +5,50 @@ const AWS = require('aws-sdk');
...
-	return Rx.Observable.fromPromise(publishToSns({
-		TopicArn: process.env.USERS_SNS_ENDPOINT,
-		Message: JSON.stringify({uuid})
-	}));
+function publishUsers(users) {
+    const records = users.map(uuid => ({
+        Data: new Buffer(JSON.stringify({ uuid })),
+        PartitionKey: 'nextmyftemailuser'
+    }));
+    console.log(records);
```
Comment (laurieboyes) [inline functions/get-users/index.js]: Can this log use n-lambda's logger instead?

File: functions/get-users/index.js [hunk 2]
```diff
@@ -5,44 +5,50 @@ const AWS = require('aws-sdk');
...
-			return Rx.Observable.merge(
-				(isLastPage) ? emptyStream() : getUsers(++page),
-				Rx.Observable.from(res)
-			);
-		});
+function getUsers(page) {
+    page = page || 0;
+        return dbQuery(page)
+            .flatMap(res => {
+                let isLastPage = res.length < queryLimit;
```
Comment (laurieboyes) [inline functions/get-users/index.js]: could be const

**Comment by laurieboyes** [comment]
👍  pedantry aside

**Pull request merged**